js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/PersistentIndexForm.tsx

```diff
@@ -0,0 +1,34 @@
+import { Stack } from "@chakra-ui/react";
+import { Form, Formik } from "formik";
+import React from "react";
+import { FormActions, IndexFormFieldsList } from "./IndexFormFieldList";
+import { useCreatePersistentIndex } from "./useCreatePersistentIndex";
+
+export const PersistentIndexForm = ({ onClose }: { onClose: () => void }) => {
```

Comment:
Need some context on each of these forms. Are they functionally the same? Just that if so, like we discussed before, we can pull a wrapper above these Index forms this up and make this more simpler. This wrapper can simply accept these hooks maybe? And also, I assume, -from the code, styling should be the same 🙂

Reply:
This is done!